docs/json_to_sarif.md

```diff
@@ -0,0 +1,151 @@
+# JSON to SARIF
+
+## Contents
+
+- [SARIF example report](#sarif-example-report)
+- [(Work in Progress) Map](#work-in-progress-map)
+
+## SARIF example report
+
+The following example illustrates how Brakeman should generate a SARIF representation of its analysis.
```

Comment:
:wave: @presidentbeef - I would love your thoughts on this proposed SARIF report for Brakeman. There are also a few [open question further down in this document](https://github.com/swinton/brakeman/blob/961ac45e42cb8624520790470bfcb185af83bdba/docs/json_to_sarif.md#questions). :bow: